Upgrade to latest react-design-tokens (#63)
component-driven/component-driven-development: src/components/primitives/Text/Text.md

> By default, it renders `<P>` element with the default text styles.

Reviewer: ```suggestion
By default, it renders `<p>` element with the default text styles.
```

```jsx
<Text>
  Lithe and leggy, Salukis are the supermodels of the canine world.
  Their looks are legendary: this ancient breed can be traced back to
  the pyramids. Archaeologists have even found these dogs entombed
  with Egyptian royalty!
</Text>
```

You can override specific attributes, such as `color` or `margin`:

```jsx
<Text color="error" margin={4}>
  This is a red text with bigger margin
</Text>
```

> but we encourage you to use `variant` prop as much as possible to ensure all text in the app look consistent:

Reviewer: ```suggestion
However, we encourage you to use the `variant` prop as much as possible to ensure consistent look of text in the app:
```
